app/controllers/compute_resources_controller.rb

```diff
@@ -112,13 +112,21 @@ def cluster_selected
     end
   end
 
+  def resource_pools
```

Comment:
API is not with nested clusters (how do I send the cluster: "NY/Manhattan/Marketing" via api?), which is why it is better to send the cluster as a parameter (not uri param)